packages/zoe/test/unitTests/contracts/test-autoswap.js

```diff
@@ -38,12 +38,13 @@ test('autoSwap with valid offers', async t => {
       TokenA: moolaR.issuer,
       TokenB: simoleanR.issuer,
     });
-    const aliceInvite = await zoe.makeInstance(
+    const bogusInvite = await zoe.makeInstance(
```

Comment:
I think we should make this an `addLiquidity` invite by default, and I think we should drop the `bogus` name.

Reply:
Done